apps/arweave/src/ar.erl

```diff
@@ -442,9 +450,15 @@ start(
 	),
 	ok = start_graphql(),
 	%% Start the first node in the gossip network (with HTTP interface).
-	ok = ar_http_iface_server:start(Port, GatewayOpts, [
+	ok = ar_http_iface_server:start([
 		{http_entrypoint_node, Node},
-		{http_bridge_node, Bridge}
+		{http_bridge_node, Bridge},
+		{port, Port},
+		{gateway, GatewayOpts},
```

Comment:
Looks like we do not need this flag now, as well as the `gateway_opts` function.

Also, when checking the port above, we could just just do `is_binary(GatewayDomain)`?

Reply:
Good catch! Though I removed it 😅 e8f9c451